Made the buttons into balls

Rovanion opened the pull request:
Den här PR:en börjar med en normal css-normalisering som inte bara sätter alla kanter, marginaler och all stoppning till null.

Vidare gör den alla knappar till små runda bollar med regn som bakgrundsbild.

Rovanion committed:
Made the buttons into balls.

Rovanion committed:
Made the buttons into balls.

Rovanion committed:
Inkludera regnbilden också.

Rovanion committed:
Inkludera regnbilden också.

majil334 committed:
Merge branch 'master' into lite-css

majil334 reviewed (dismissed):
Snyggt! Nu när master är mergad måste dock de högra knapparna (turkey, world osv) flyttas uppåt så man kan trycka på alla knappar. 

Man kan inte läsa texten på alla knappar heller, är väl bara "Temperature" som det spelar roll på iof!

MangeI89 commented on resources/public/css/style.css:
```diff
@@ -87,23 +128,22 @@ input[type=range] {
 }
 
 .button:hover {
-  box-shadow: 0 0 50px #255aaa;
```
Tycker att vi ska ha blå skugga (istället för svart) när man hovrar, annars får man ingen indikation att man kan klicka på knappen.

Rovanion replied:
Ordnat!

MangeI89 reviewed (changes requested):
Tycker formen på knapparna var bättre innan, iaf för de som ligger högst upp. Tycker även att blå skugga blir bättre än svart, se enskild kommentar.

Rovanion committed:
Implemented MangeI's suggestion on box shadow color.

Rovanion committed:
Implemented MangeI's suggestion on box shadow color.

Rovanion committed:
Merge branch 'master' of github.com:WeatherMagic/weather-front into lite-css

Rovanion committed:
Made almost everything beautiful.

Rovanion committed:
Made almost everything beautiful.

Rovanion committed:
Now with appropriate cursors, and no text can be selected no more.

The css file is also sorted alphabetically and everything is aligned, somewhat.

Rovanion committed:
Now with appropriate cursors, and no text can be selected no more.

The css file is also sorted alphabetically and everything is aligned, somewhat.

Rovanion committed:
Put the UI inside the canvases so that the user can drag even through ui elements.

Rovanion committed:
Put the UI inside the canvases so that the user can drag even through ui elements.

Rovanion committed:
Landing page now looks better on screens of almost all sizes.

Rovanion committed:
Landing page now looks better on screens of almost all sizes.

Rovanion commented:
Förhoppningsvis har alla era drömmar och förhoppningar, farhågor och orosmåln blivit besvarade i och med de senaste commitsen.

Jag får medge att standardknappen är ganska ful. Men kompassen, den ni, den är så vacker att man kan gråta.

majil334 commented:
Sjukt snygg kompass ja!! 👍 

Dock saknar jag att jorden snurrar i initialt läge. Knappen för detta också.

Såg att när man drar i time-slidern så snurras jorden dock, det är ju fel?

Personligen tycker jag att det vore nice med lite mer glow när man hovrar över knappar också :)

Rovanion committed:
Merge branch 'master' of github.com:WeatherMagic/weather-front into lite-css

Rovanion committed:
Fixed cursor bugs, moved ui out of canvases and added larger button glow.

Rovanion committed:
Fixed cursor bugs, moved ui out of canvases and added larger button glow.

Commit:
Merge branch 'master' into lite-css

Rovanion committed:
Merge branch 'lite-css' of github.com:WeatherMagic/weather-front into lite-css

Rovanion commented:
@majil334 Fixade dina påpekanden. Tyvärr innebär det att dragningen avbryts när man drar musen över gränssnittselement.

MangeI89 committed:
Merge branch 'lite-css' of github.com:weathermagic/weather-front into lite-css

MangeI89 committed:
Merge branch 'master' of github.com:weathermagic/weather-front into lite-css

MangeI89 committed:
Merge with master

MangeI89 committed:
Merge branch 'lite-css' of github.com:weathermagic/weather-front into lite-css

MangeI89 committed:
Checkedout correct earth-img from master
